content/posts/android-v2.1.0.md

```diff
@@ -0,0 +1,52 @@
+---
+title: 'Android v2.1.0 - Jellyfin in your car!'
+subtitle: 'Listen to your music while driving with the new Android update'
+author: 'Niels van Velzen'
+githubusername: 'nielsvanvelzen'
+date: 2020-10-25T00:00:00-00:00
+justify: 'center'
+---
+
+Jellyfin for Android version 2.1 is here with support for Android Auto.
+
+<!--more-->
+
+A new version of the Android app: version 2.1 has just been released! This update contains some quality of life improvements, some bugfixes and new features.
+It's been only a few weeks since 2.0 but we couldn't wait to bring these new exiting changes to you. Let's start with the most awesome feature.
+
+## Android Auto
+
+Starting with this update we now support playing music in cars. This change was contributed by [@Spacetech](https://github.com/Spacetech) and we are excited to release it. Right now it supports browsing your music library in multiple categories: latest, albums, artists, songs and genres. It allows shuffling your albums and shows thumbnails when available.
+
+<video controls loop autoplay muted playsinline class="inline justify" height="500">
+	<source src="/images/posts/android-2-1-0/android-auto.webm" type="video/webm" />
+</video>
+
+___Note:__ since we do not have offline-support at this moment all music playback needs an active network connection to work. Be aware this may impact your mobile internet bundle._
```

Comment:
```suggestion
_**Note:** since we do not have offline-support at the moment all music playback needs an active network connection to work. Be aware this may cause additional charges in your mobile plan._
```